src/olympia/amo/models.py

```diff
@@ -26,6 +27,21 @@
 log = olympia.core.logger.getLogger('z.addons')
 
 
+@Field.register_lookup
+class Like(Lookup):
```

Comment:
As this lookup is going to be available to all fields can we have a generic test for it under `amo/tests/`?